### DISCUSSION TITLE
Wine

### DISCUSSION THREAD

**Discussion by AdrianKrajnik** [Vanilla-OS/live-iso#382] [Q&A] [answered]
Hi,
I have many problems with Wine on Vanilla OS 2. Every time I try to use Wine, it says: 'It looks like wine32 is missing. You should install it. As root, please execute: apt-get install wine32:i386.' I tried running apt-get install wine32:i386 as root, but it says: 'Cannot find package wine32:i386. 
Can someone help me?

**Comment by taukakao** [comment, marked as answer]
We don't ship 32 bit packages in our repos anymore. 
If you want low level access through the terminal I recommend using an apx container, if you prefer a graphical interface then I would recommend Bottles or Lutris.
